noxfile.py

```diff
@@ -20,25 +116,30 @@
 
 html = ("-b", "html")
 check_hyperlinks = ("-b", "linkcheck", "-q")
+documentation_requirements = get_requirements_file(category="docs", version=maxpython)
 
 
 @nox.session(python=maxpython)
 def docs(session):
     """Build documentation with Sphinx."""
-    session.install(*doc_requirements)
+
+    session.install("-r", documentation_requirements)
     session.install(".")
     session.run(*sphinx_commands, *html, *session.posargs)
 
 
 @nox.session(python=maxpython)
 def linkcheck(session):
     """Check hyperlinks in documentation."""
-    session.install(*doc_requirements)
+    session.install("-r", documentation_requirements)
     session.install(".")
     session.run(*sphinx_commands, *check_hyperlinks, *session.posargs)
 
 
-@nox.session(python=maxpython)
```

Comment:
Not specifying the version of Python will make it so that contributors can locally run mypy with whatever compatible version of Python they have installed.